scripts/test/compactionPerformance/system-test-instance.properties

```diff
@@ -65,52 +65,10 @@ sleeper.optional.stacks=SystemTestStack,CompactionStack,IngestStack
 # Whether to keep the sleeper table bucket, Dynamo tables related to tables and query results bucket or not.
 sleeper.retain.infra.after.destroy=false
 
-# The maximum number of concurrent compaction tasks to run
-sleeper.compaction.max.concurrent.tasks=300
-
-# Partitions with more than the following number of records in will be split
-# Set to 4 billion so that one round of splitting compactions happen
-sleeper.default.partition.splitting.threshold=4000000000
-
-# How often partition splitting should happen
-sleeper.partition.splitting.period.minutes=10
-
-# The maximum number of files in a partition splitting job
-sleeper.partition.splitting.files.maximum=50
-
-# The frequency with which the garbage collector is run
-sleeper.gc.period.minutes=5
-
-# The size of the batch of files that are ready for garbage collection requested from the State Store
-sleeper.gc.batch.size=2000
-
-# The length of time after a message is consumed from a queue before it becomes visible again
-sleeper.queue.visibility.timeout.seconds=900
-
-# The frequency with which a compaction job sends keep alive messages to the job queue
-sleeper.compaction.keepalive.period.seconds=600
-
-# The rate at which the compaction job creation lambda runs (in minutes, must be >=1)
-sleeper.compaction.job.creation.period.minutes=1
-
-# The rate at which a check to see if compaction ECS tasks need to be created is made (in minutes, must be >= 1)
-sleeper.compaction.task.creation.period.minutes=1
-
-# The amount of memory for the lambda that creates compaction jobs
-sleeper.compaction.job.creation.memory=1024
-
-# The amount of memory for the lambda that runs ECS tasks to execute compaction jobs
-sleeper.task.runner.memory=1024
-
-# The timeout for the lambda that creates compaction jobs in seconds
-sleeper.compaction.job.creation.timeout.seconds=900
-
-# The timeout for the lambda that creates compaction jobs in seconds
-sleeper.task.runner.timeout.seconds=900
-
-# The CPU architecture to run compaction tasks on.
-# See Task CPU architecture at https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html
-sleeper.compaction.task.cpu.architecture=X86_64
```

Comment:
Looks like this property was missed when setting the resources and architecture explicitly. It's also probably not necessary to set the resources for the architecture we're not using.